apps/backend/cloudflare/src/lib/db/generate_schemas.ts

```
@@ -0,0 +1,220 @@
+
+  return lines.join('\n');
+};
+
+// ── Main ─────────────────────────────────────────────────────────────────
+
+const main = async (): Promise<void> => {
+  const args = process.argv.slice(2);
+  const isCheck = args.includes('--check');
+
+  if (isCheck) {
+    console.log('Checking generated row schemas are up to date…');
```

Biome (2.5.8)

[error] Don't use console.

(lint/suspicious/noConsole)